src/mtype.d

```diff
@@ -8564,39 +8551,24 @@ public:
         Dsymbol s;
         static if (LOGDOTEXP)
         {
-            printf("TypeClass::dotExp(e='%s', ident='%s')\n", e.toChars(), ident.toChars());
-        }
-        if (e.op == TOKdot)
```

Comment:
`DotExp` is already handled in `DotIdExp.semanticY`, so this code is unreachable today.